pkgs/applications/misc/pagefind/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, callPackage
+, rustPlatform
+, fetchFromGitHub
+, fetchNpmDeps
+, npmHooks
+, binaryen
+, gzip
+, nodejs
+, wasm-bindgen-cli
+, wasm-pack
+}:
+
+let
+
+  # https://github.com/NixOS/nixpkgs/issues/89426
+  # we directly re-use the existing fix to avoid adding
+  # more rustc versions to the cache should the fixes diverge
+  rustc-wasm = callPackage ./../../../servers/ldap/lldap/rustc-wasm.nix { };
+
+  wasm-bindgen-84 = wasm-bindgen-cli.override {
+    version = "0.2.84";
+    hash = "sha256-0rK+Yx4/Jy44Fw5VwJ3tG243ZsyOIBBehYU54XP/JGk=";
+    cargoHash = "sha256-vcpxcRlW1OKoD64owFF6mkxSqmNrvY+y3Ckn5UwEQ50=";
+  };
```

Comment:
I would expect `overrideAttrs` here

Reply:
Intuitively i would agree, but updating hashes in fetchers is a chore. `.override` was enabled here: https://github.com/NixOS/nixpkgs/commit/b6910512a0789390af9db312ee8f08665bd662ab